db/src/db.rs

```diff
@@ -178,13 +306,42 @@ mod tests {
             .prefix("test_set_rocksdb_options")
             .tempdir()
             .unwrap();
+        let options: HashMap<String, String> = toml::from_str(
```

Comment:
There are a lot of configurations, and the comments in RocksDB source code is not correct for all. So, I test those settings one by one. 
After test them in `ckb.toml`, I don't want to re-format them manually again, and I think if I leave them in the unit test then people can copy them to use easily in future.
So, I list all settings which I have test and keep them in this unit test, otherwise, I can just keep this test like before.